### DISCUSSION TITLE
Trying to update to the latest version

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2455] [Help] [answered]
Hello,

When I try to update BB I get the message that I already have the latest version but my current version is 0.9.111, so I am trying to update the image with cli:

**First try**
![image](https://user-images.githubusercontent.com/53205779/130690201-82fe0b62-5d2b-4d7f-b292-46a0851f0f46.png)

**Getting new yaml files**
![image](https://user-images.githubusercontent.com/53205779/130690014-a766a264-23e8-4962-a104-455b88069ba2.png)

**Same error**
![image](https://user-images.githubusercontent.com/53205779/130690136-f377f0a6-9887-42d1-8271-026c90c19040.png)

**Trying to use docker-compose pull**
![image](https://user-images.githubusercontent.com/53205779/130690321-17b1aef1-54fe-4426-8717-41448808b5a1.png)

**Comment by shogunpurple** [comment]
Hi @berma83 - You seem to be missing a port entry in your `.env` file. Make sure that your `.env` file has all the following defined:

```
APP_PORT=4002
WORKER_PORT=4003
MINIO_PORT=4004
COUCH_DB_PORT=4005
REDIS_PORT=6379
WATCHTOWER_PORT=6161
BUDIBASE_ENVIRONMENT=PRODUCTION
```

Then run `docker-compose up -d`

**Reply by berma83** [reply]
Hi @shogunpurple, thank you for your help.

My .env looks ok.

```
# Use the main port in the builder for your self hosting URL, e.g. localhost:10000
MAIN_PORT=10000

# This section contains variables that do not need to be altered under normal circumstances
APP_PORT=4002
WORKER_PORT=4003
MINIO_PORT=4004
COUCH_DB_PORT=4005
REDIS_PORT=6379
WATCHTOWER_PORT=6161
BUDIBASE_ENVIRONMENT=PRODUCTION
```

**Comment by berma83** [comment, marked as answer]
Found the problem!

My docker-compose.yaml was in html format.

If I use the link below to download, the file comes in html format:
`wget https://github.com/Budibase/budibase/blob/develop/hosting/docker-compose.yaml`

_type: HTML document, UTF-8 Unicode text, with very long lines_


The correct link is:
`wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml`

_type: ASCII text_


Sorry, my bad.
